src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Lottery ball = new Lottery();
```

Comment:
don't create new Lottery in for loop
bad naming - ball

Reply:
How I need to create Lottery objects and how balls need to be named?